modules/trimgalore/main.nf

```diff
@@ -11,12 +11,13 @@ process TRIMGALORE {
     tuple val(meta), path(reads)
 
     output:
-    tuple val(meta), path("*.fq.gz")    , emit: reads
-    tuple val(meta), path("*report.txt"), emit: log
-    path "versions.yml"                 , emit: versions
+    tuple val(meta), path("${outprefix}"), emit: reads
```

Comment:
Thoughts on dropping `outprefix` for? Just curious

```
tuple val(meta), path("*{trimmed,val}*.fq.gz"), emit: reads
```

Reply:
I thought `outprefix` was more explicit but this works too 👍🏽 
https://github.com/nf-core/modules/pull/1568/commits/e6cefebf1b3a0d284dec82d30820eb6caef348dc